grafana-plugin/src/containers/RotationForm/ScheduleOverrideForm.tsx

```diff
@@ -172,7 +166,7 @@ const ScheduleOverrideForm: FC<RotationFormProps> = (props) => {
     setErrors({});
 
     store.scheduleStore
-      .updateRotationPreview(scheduleId, shiftId, startMoment, true, params)
+      .updateRotationPreview(scheduleId, shiftId, store.timezoneStore.calendarStartDate, true, params)
```

Comment:
I thought you **always** do it in the following way `const {timezoneStore: {calendarStartDate}} = useStore();` =)

Reply:
I do it when we already have destructuring from `store`, seems that here we don't (e.g. the line above we have `store.scheduleStore`, not `scheduleStore`). This PR is already quite big so to make changes more understandable I decided to not refactor those places and follow whatever is done in respective components:) But in general yes you're right - I think destructuring the store is a good thing, I just didn't want to refactor too much in one PR

Reply:
ok, sounds good